docs/tutorials/scala/mxnet_scala_on_intellij.md

```diff
@@ -347,6 +379,30 @@ Library not loaded: /usr/local/opt/opencv/lib/libopencv_calib3d.x.x.dylib
 This can be resolved be installing OpenCV.
 
 
+### Using MXNet from source
+
+If you chose to "Build from Source" when following the [install instructions](https://mxnet.incubator.apache.org/install/index.html), you can use your custom build instead of the build from maven.  Use your build by editing the `pom.xml` file and replacing the `org.apache.mxnet` dependency with the following:
+
+```
+      <groupId>org.apache.mxnet</groupId>
+      <artifactId>mxnet-core_${scala.version}-${platform}-sources</artifactId>
+      <version>1.3.0</version>
```

Comment:
maybe skip this?

Reply:
Yeah, don't have to include this here...